addons/microdagr/functions/fnc_deviceAddWaypoint.sqf

```diff
@@ -17,8 +17,8 @@
  * Public: No
  */
 
-params ["_waypointName","_waypointPosASL"];
+params ["_waypointName", "_waypointPosASL"];
 
 private _waypoints = ACE_player getVariable [QGVAR(waypoints), []];
-_waypoints pushBack [_waypointName, _waypointPosASL];
+GVAR(currentWaypoint) = _waypoints pushBack [_waypointName, _waypointPosASL];
```

Comment:
Maybe only switch if we don't have an active waypoint?

```suggestion
private _new = _waypoints pushBack [_waypointName, _waypointPosASL];
if (GVAR(currentWaypoint) == -1) then { GVAR(currentWaypoint) = _new };
```

Reply:
I agree I'll be reworking this. I think setting a WP as active shall be a decision of the user. We can make it easier by preselecting the newly created waypoint.